quickstarts/ocs-install-tour-quickstart.yaml

```diff
@@ -0,0 +1,62 @@
+  description: Install the OpenShift Container Storage (OCS) operator and create a storage cluster.
+  introduction: Red Hat OpenShift Container Storage is persistent software-defined storage integrated with and optimized for Red Hat OpenShift Container Platform. Dynamic, stateful, and highly available container-native storage can be provisioned and de-provisioned on demand as an integral part of the OpenShift administrator console.
+  tasks:
+  - title: Installing the OpenShift Container Storage
+    description: |- 
+      The OperatorHub is where you can find a catalog of available Operators to install on your cluster.
+      
+      Follow these steps to Install OpenShift Container Storage
+
+      1. In the navigation menu, select **Operators > OperatorHub**
+      2. In the search bar, enter  **OpenShift container storage**
+      3. Click the **OpenShift Container Storage** tile to initiate the operator installation on your cluster.
+      4. Click **install** when you're finished with Operator Subscription form.
+      5. When installation is finished click **Create Storage Cluster**.
+    review:
+      instructions: |-
+        ####  To verify that the OpenShift Container Storage operator is installed. 
+
+        Does the status of the installed operator show as Succeeded?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have installed the OpenShift Container Storage operator!
+      failed: Try the steps again.
+  - title: Create a storage cluster
+    description: |-
+      A storage cluster includes a Ceph cluster, Multi cloud gateway and all the storage and commute resources required.
+
+      Follow these steps to create a storage cluster
+
+      1. Select the cluster mode
+          - **Internal** - Can be used on any platform except bare metal.
+          - **Internal-Attached device**- Can be used on any platform where there are attached devices to the nodes, using the Local Storage Operator (LSO)
+          - **External**- For external RHCS Cluster
+      2. Follow the instructions on the page to create the storage cluster.
+      3. Once the storage cluster is created you can start using the OpenShift container storage resources (storage classes and the bucket classes) to allow OCS to manage your storage.
+    review:
+      instructions: |-
+        #### To verify that the Storage Cluster is created.
+
+        Check and see if the status of the storage cluster is in a ready state?
```

Comment:
On the storage cluster details page, is the state of the cluster **Ready** ?

Reply:
done